Form/Type/AddressEmbeddableType.php

```diff
@@ -74,11 +77,13 @@ public function configureOptions(OptionsResolver $resolver)
             'allowed_countries' => [], // After updating to Symfony 5.1, make this a filter list based on country code.
             'preferred_countries' => [],
             'default_country' => null,
+            'choice_translation_domain' => null,
```

Comment:
The default in ChoiceType is true, why make the default TRUE here? See https://symfony.com/doc/current/reference/forms/types/choice.html#choice-translation-domain. If you make the default true here, you don't need the isset higher up and can just add it.

Reply:
Well, in the doc, the default value is TRUE, but in the code, it is FALSE, isn't it? https://github.com/symfony/symfony/blob/6.1/src/Symfony/Component/Form/Extension/Core/Type/CountryType.php#L43

That's why i did that way

Reply:
Yes, I was looking at the wrong base class, you're absolutely right!

Reply:
Well, the default should still be the same afterwards though, meaning the default here should be false then. If you do that you follow the standards from above and you can remove the isset (see other comment)